### PR TITLE
Update on the final page

### PR DESCRIPTION
**Pull request by vigehi**
Update the final SPA of the  calculator

### CODE REVIEW

File: src/js/header.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <h1>Edith Magic</h1>
+    <ul>
+      <a href="/">Home</a>
+      <a href="/Calculator">Calculator</a>
+      <a href="/Quotes">Quotes</a>
+    </ul>
+  </header>
+);
+
+export default Header;
```
Comment (1GORDON) [inline src/js/header.js]: Great job @vigehi 🤝

- [x] Kindly maintain one `header.js` file. Your project appears to be having two similar files. You may remove one and keep the other.

File: src/js/header.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './css/header.css';
+
+const Header = () => (
+  <header>
+    <h1>Edith Magic</h1>
+    <ul>
+      <a href="/">Home</a>
+      <a href="/Calculator">Calculator</a>
```
Comment (1GORDON) [inline src/js/header.js]: Great job @vigehi 🤝

- [x] Kindly take a closer look at the screen of your calculator when you perform a mathematical calculation. The entries seem to be hidden and then the final result is displayed. Kindly make sure the transactions are displayed before the final result is displayed.

calculator

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @vigehi ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/header.js (hunk 1 above)
Comment (Whoistolu) [reply src/js/header.js]: Well resolved 👍🏽

File: src/js/header.js (hunk 2 above)
Comment (Whoistolu) [reply src/js/header.js]: Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @vigehi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**